go.mod

```diff
@@ -1,13 +1,20 @@
 module github.com/weaveworks/flux-adapter
 
-go 1.12
+go 1.15
```

Comment:
Is this a good idea if it's still building at CircleCI with Go 1.12?

Reply:
Yar, I bumped this without thinking too hard. New commit pushed; if it builds, there's your answer :-)